readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -468,3 +468,79 @@ def test_unicode(self):
             content_type='application/json',
         )
         self.assertEqual(resp.status_code, 200)
+
+    def test_user_defined_stable_version_with_tags(self):
+
+        Version.objects.create(
+            project=self.pip,
+            identifier='0.8.3',
+            verbose_name='0.8.3',
+            active=True,
+        )
+
+        Version.objects.create(
+            project=self.pip,
+            identifier='foo',
```

Comment:
`origin/stable` here I guess.

Reply:
I'm tested it gets updated from a prior "set" version.